crates/bevy_render/src/view/window/mod.rs

```diff
@@ -420,12 +387,15 @@ pub fn prepare_windows(
     }
 }
 
-pub fn need_new_surfaces(
+pub fn need_surface_configuration(
```

Comment:
This is technically a breaking change. Might need to document this.

Reply:
I would be very curious of anyone actually using that, but mentioned it!

Reply:
https://github.com/search?q=need_new_surfaces+NOT+is%3Afork&type=code

I don't think anyone is using it, which is probably why it's fine to put into 0.13.1, but that's still technically breaking semver.